app/js/l10n.js

```diff
@@ -0,0 +1,41 @@
+(function() {
+  function format(string, formatKwargs) {
+    if (formatKwargs === undefined) {
+      return string;
+    }
+
+    return string.replace(/{[^}]+}/, function(input) {
```

Comment:
This should be `/{[^}]+}/g` to match several arguments. I've noticed this when you click on a top helper, the numbers of questions solved and helpful votes displays as {n}.